components/area_office_table.js

```diff
@@ -60,7 +60,7 @@ const pin = css`
   font-size: 30px !important;
 `;
 const provinceCell = css`
-  color: ${globalTheme.colour.white};
+  color: ${globalTheme.colour.greyishBrown};
```

Comment:
lol how'd that happen? 
oh whoops, in https://github.com/cds-snc/vac-benefits-directory/commit/b4d194d5311a068b0fe9f38db99aff65ce3a9b0e#diff-de8d82b8648719a2685e77b28f843151L62 Adam changed a `#000` to white (and I approved the PR, dooh!)